roscala/src/main/scala/coop/rchain/rosette/prim/Tuple.scala

```diff
@@ -0,0 +1,37 @@
+package coop.rchain.rosette.prim
+
+import coop.rchain.rosette.{
+  Ctxt,
+  Fixnum => RFixnum,
+  Ob,
+  RblBool,
```

Comment:
You don't need that here.

Reply:
I'll take out the rename, but leave Fixnum because I'll need it later.